kotlinx-coroutines-core/common/test/DurationToMillisTest.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2016-2023 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+package kotlinx.coroutines
+
+import kotlin.test.*
+import kotlin.time.*
+import kotlin.time.Duration.Companion.milliseconds
+import kotlin.time.Duration.Companion.nanoseconds
+import kotlin.time.Duration.Companion.seconds
+
+class DurationToMillisTest {
+
+    @Test fun negative_duration_coerced_to_zero_millis() = assertEquals(
+        expected = 0L,
+        actual = (-1).seconds.toDelayMillis(),
+    )
```

Comment:
Style: please look at how all the other tests are structured in this library.
* Test names: we use camel case, and also, all test names begin with `test`.
* Indentation is different: `@Test` should be on a separate line, function calls are typically on a single line, etc.
* Explicit parameter names here don't add much and are mostly noise.